components/log-viewer-webui/server/settings.json

```diff
@@ -8,9 +8,9 @@
     "MongoDbName": "clp-query-results",
     "MongoDbStreamFilesCollectionName": "stream-files",
 
-    "ClientDir": "../client/dist",
-    "LogViewerDir": "../yscope-log-viewer/dist",
-    "StreamFilesDir": "../../../build/clp-package/var/data/streams",
+    "ClientDir": "../../client/dist",
+    "LogViewerDir": "../../yscope-log-viewer/dist",
+    "StreamFilesDir": "../../../../build/clp-package/var/data/streams",
```

Comment:
It seems we still need to add one more level up (i.e., an additional `../`) per path to get the paths resolved correctly.

Also, the `.` current directory is assumed to be where `settings.json` located.

I believe we should change `rootDirname` -> `rootDirname = path.resolve(dirname, "../../../..")` in `src/routes/static.ts`, given where the `static.js` is in `dist` now.

@coderabbitai create an issue to track this